src/Train Toxicity Model.ipynb

```diff
@@ -367,22 +385,39 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "hi\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('hi')"
```

Comment:
Looks like you have a stray `print('hi')` cell here.

Reply:
Good point. Removed in most recent commit.